### DISCUSSION TITLE
Nuxt 3: npm - critical vulnerabilitiy

### DISCUSSION THREAD

**Discussion by rudimadima** [nuxt/nuxt#17266] [Questions] [answered]
When installing the Nuxt 3 app, npm wrote a message:
```
found 8 vulnerabilities (2 moderate, 5 high, 1 critical) in 681 scanned packages
```
I run `npm audit`:
```
  Critical        Server-Side Request Forgery in parse-url                      
  Package         parse-url                                                     
  Dependency of   nuxt [dev]                                                    
  Path            nuxt > @nuxt/telemetry > git-url-parse > git-up > parse-url   
  More info       https://github.com/advisories/GHSA-7f3x-x4pr-wqhj     
```
Could it be dangerous?

**Comment by danielroe** [comment, marked as answer]
As the security issue in `parse-url` was fixed in a patch release, you only have to run `yarn upgrade` (or npm/pnpm equivalent) in your project to benefit from the update. Likewise, any new Nuxt projects will be using the newer version automatically.

Moreover, Nuxt is not affected by this vulnerability. The URL that is parsed is from the developer's machine, and even if any remote URLs are wrongly parsed, it will just change the internal project hash.

https://github.com/nuxt/framework/discussions/5780#discussioncomment-3109485

**Reply by amrnn90** [reply]
Hey @danielroe, 
Unfortunately this problem still exists even on a fresh project using the latest `nuxt@3.0.0-rc.6`.
